packages/react-client/src/hooks/devices/useCamera.ts

```diff
@@ -3,59 +3,65 @@ import type { DeviceItem, TrackMiddleware } from "../../types/public";
 import { useDeviceApi } from "../internal/device/useDeviceApi";
 import { useFishjamContext } from "../internal/useFishjamContext";
 
-type CameraApi = {
+/**
+ * @category Devices
```

Comment:
Adding `category` for types, make it less readable. As they will now appear together with functions on docs website. 
![Screenshot 2025-01-15 at 17 41 56](https://github.com/user-attachments/assets/36271dd4-5533-4f37-9e76-cc68481f589c)